pallets/liquidity-pools-gateway-queue/src/lib.rs

```diff
@@ -93,6 +98,11 @@ pub mod pallet {
 			message: T::Message,
 			error: DispatchError,
 		},
+
+		/// Maximum number of messages was reached.
+		MaxNumberOfMessagesWasReached {
```

Comment:
Nit: Would remove `Was`  but absolutely non-blocking!

```suggestion
		MaxNumberOfMessagesReached {
```

Reply:
Will change.